redisai/commands.go

```diff
@@ -250,3 +250,8 @@ func AddDagRunArgs(loadKeys []string, persistKeys []string, commandArgs redis.Ar
 	}
 	return args
 }
+
+// Sets the default backends path
+func (c *Client) SetBackendPath(path string) (string, error) {
```

Comment:
@dengliming should we set it to plural ( setBackendsPath )? given that the config property is BACKENDSPATH